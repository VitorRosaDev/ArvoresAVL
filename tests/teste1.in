i 45
i 20
i 60
em
pre
pos
sair