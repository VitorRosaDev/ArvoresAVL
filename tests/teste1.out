OK: inserido 45
Árvore:
└── 45
OK: inserido 20
Árvore:
└── 45
    └── 20
OK: inserido 60
Árvore:
└── 45
    ├── 20
    └── 60
Em-Ordem: 20 45 60
Pré-Ordem: 45 20 60
Pós-Ordem: 20 60 45
Encerrado.